notebooks/data-sources/oc-github-repo/time_to_merge_model.ipynb

```diff
@@ -0,0 +1,1277 @@
+{
```

Comment:
<p>Can we add labels for x and y axes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>sure, will do 👍</p>

Reply:
added axes :)